README.md

```diff
@@ -260,8 +261,13 @@ filegroup(
     name = "node_modules",
     srcs = glob(
         include = ["node_modules/**/*"],
-        # Files with spaces in the name are not legal Bazel labels
         exclude = [
+          # Files under test & docs may contain file names that
+          # are not legal Bazel labels (e.g.,
+          # node_modules/ecstatic/test/public/中文/檔案.html)
+          "node_modules/test/**",
+          "node_modules/docs/**",
+          # Files with spaces in the name are not legal Bazel labels
```

Comment:
should we remove the space exclusions now that we require bazel 0.17?

Reply:
Are those legal now? I didn't see that in the Bazel release notes. I'll test it.